### DISCUSSION TITLE
Eyeball mk1 on support units

### DISCUSSION THREAD

**Discussion by Parel803** [PygmalionOfCyprus/cmo-db-requests#3879] [General]
Good evening,

Just a question about the Mk1 on 'ship' supporting units like oil slick, ice berg, mooring bouy, lost shipping container, marker invisible, floating drugs, floating wreckage.
And maybe the target training ship, I fired on those. There are cameras but :-) Or is not intended as a ship to fire on? 
Same Q for the Saildrone USV. Or do I just misundestand the eyeball Mk1 in CMO than nevermind me and apologize.
Are these mend to have this sensor? If not I try to add to the issue page.

best regards GJ

**Comment by PygmalionOfCyprus** [comment]
We can't define those DB-side; it's defined in-code which platform types have the Mk1 Eyeball. The None, Unknown, etc. types seem to get it by default. Might be worth throwing a request in the forums that ships of Category None get their eyeballs gouged out. (I checked, all the None ships are unmanned).

**Comment by Parel803** [comment]
Thanks. I'll make the request.
regards GJ
